devicetypes/samsungsds/samsung-smart-doorlock.src/samsung-smart-doorlock.groovy

```diff
@@ -351,31 +349,31 @@ private def parseCommandResponse(String description) {
 		def alarmCode = Integer.parseInt(data[0], 16)
 		switch (alarmCode) {
 			case 0: // Deadbolt Jammed
```

Comment:
wouldn't a block commenting out this whole method be better here?

Reply:
Good suggestion, I comment out this whole block